lwt/core_lwt.ml

```diff
@@ -0,0 +1,30 @@
+open Core_kernel.Std
```

Comment:
Honest question: What is the behavior of Lwt if a cooperative thread performs a blocking syscall? (I really don't know, all I know about Lwt is that it can't use multiple cores and uses cooperative threading, which normally has a problem with this.)

Worry cases:
1.) Bap as a service - need an entirely separate bap instance for every connection to avoid accidental DOS on one thing blocking.
2.) Bap server hardlocking on an operation (e.g. file open or read from a network share?) and being unable to report to the client.